src/lib.rs

```diff
@@ -38,17 +38,20 @@ fn encode(current: f64, previous: f64, factor: i32) -> Result<String, String> {
     Ok(output)
 }
 
-/// Encodes a Google Encoded Polyline.
+/// Encodes a Google Encoded Polyline. Accepts a Vec or slice
```

Comment:
It actually just accepts a `slice`. Since a `Vec` can [`Deref`](https://doc.rust-lang.org/book/deref-coercions.html) to a `slice`, the `Vec` is treated as a `slice`. Anything that can `Deref` to a `slice` (or a `slice` itself) can be passed as an argument.